src/components/common/UserMenu.jsx

```diff
@@ -0,0 +1,110 @@
+// import { Avatar, Chip, Typography, Box, Menu, MenuItem, ListItemIcon , Logout } from '@mui/material';
```

Comment:
Do we need this comment going forward?

Reply:
Good point. Thanks for catching it. 
Usually when things are from the same library, we can import as single line. 
e.g import {a,b.c} from 'library'
but somehow I got errors (conflict with Next.js), so I googled it, and someone suggested to separate them. I still don't know the rational behind. but it worked.

Reply:
The import above is named items, but the ones below are unnamed default values.  Probably the name of the items don't match exactly, which would explain why you were getting an error.  This is common for MUI Icons.